### PR TITLE
439 show breadcrumb for manufacturer

### PR DESCRIPTION
**Pull request by UlrichKu**
| Q             | A
| ------------- | ---
| Branch        | 6.2.x
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no - minor: will change breadcrumb/check for manufacturer
| Deprecations? | no
| Tests pass?   | yes
| Fixed issues  | chameleon-system/chameleon-system#439
| License       | MIT

### CODE REVIEW

File: src/ShopBundle/Interfaces/ShopServiceInterface.php [hunk 1]
```diff
@@ -81,4 +81,9 @@ public function getActiveCategory();
     public function getActiveBasket();
 
     public function resetBasket();
+
+    /**
+     * @return null|\TdbShopManufacturer
+     */
+    public function getActiveManufacturer();
```
Comment (rnuernberg) [inline src/ShopBundle/Interfaces/ShopServiceInterface.php]: Interface changes are not permitted in patch level releases

File: src/ShopBundle/objects/WebModules/MTShopBreadcrumbCore/MTShopBreadcrumbCore.class.php [hunk 2]
```diff
@@ -45,23 +48,50 @@ public function &Execute()
             }
 
             foreach (array_keys($aList) as $index) {
-                $oBreadCrum->AddItem($aList[$index]);
+                $breadcrumb->AddItem($aList[$index]);
             }
 
-            $this->data['oBreadcrumb'] = &$oBreadCrum;
+            $this->data['oBreadcrumb'] = &$breadcrumb;
```
Comment (rnuernberg) [inline src/ShopBundle/objects/WebModules/MTShopBreadcrumbCore/MTShopBreadcrumbCore.class.php]: if this code is being changed anyway, you might as well remove the `&`

File: src/ShopBundle/objects/WebModules/MTShopBreadcrumbCore/MTShopBreadcrumbCore.class.php [hunk 3]
```diff
@@ -72,19 +102,16 @@ public function _GetCacheParameters()
             $aParameters['iactiveitemid'] = $oActiveItem->id;
         }
 
+        $activeManufacturer = $this->getShopService()->getActiveManufacturer();
+        if (null !== $activeManufacturer) {
+            $aParameters['activemanufacturer'] = $activeManufacturer->id;
```
Comment (rnuernberg) [inline src/ShopBundle/objects/WebModules/MTShopBreadcrumbCore/MTShopBreadcrumbCore.class.php]: Add `id` (`activemanufacturerid`)

File: src/ShopBundle/objects/WebModules/MTShopBreadcrumbCore/MTShopBreadcrumbCore.class.php (hunk 2 above)
Comment (UlrichKu) [reply src/ShopBundle/objects/WebModules/MTShopBreadcrumbCore/MTShopBreadcrumbCore.class.php]: ok

File: src/ShopBundle/objects/WebModules/MTShopBreadcrumbCore/MTShopBreadcrumbCore.class.php (hunk 3 above)
Comment (UlrichKu) [reply src/ShopBundle/objects/WebModules/MTShopBreadcrumbCore/MTShopBreadcrumbCore.class.php]: ok

File: src/ShopBundle/Interfaces/ShopServiceInterface.php (hunk 1 above)
Comment (UlrichKu) [reply src/ShopBundle/Interfaces/ShopServiceInterface.php]: ok, moved to a private method in the module

**Pull request merged**